src/components/Calendar/Calendar.tsx

```diff
@@ -76,7 +76,7 @@ export const Calendar: FC<CalendarProps> = ({ data, onItemClick, topBarWidth })
   const handleSearch = (event: ChangeEvent<HTMLInputElement>) => {
     setSearchPhrase(event.target.value);
     debouncedFilterData.current.cancel();
-    debouncedFilterData.current(event.target.value);
+    debouncedFilterData.current(data, event.target.value);
```

Comment:
Can we put `event.target.value` into a constant on the top of this function? It's used twice - here and in `setSearchPhrase` and it would be more obvious what we set there if it has its own name, e.g. something like this:
```typescript
const handleSearch = (event: ChangeEvent<HTMLInputElement>) => {
    const phrase = event.target.value;
    setSearchPhrase(phrase);
    debouncedFilterData.current.cancel();
    debouncedFilterData.current(data, phrase);
```
What do you think?

Reply:
Changed